numpy-cheatsheet/cheatsheet-generator.py

```diff
@@ -0,0 +1,98 @@
+# generate_notebook.py
```

Comment:
> You mentioned that you are using Groq instead of Nebari to generate cheatsheets. So are you using this script at all? If you're not using it and refining it, then I don't see any need to include it here.
> 
> I do think you could include another file that shares how you generated the cheatsheets. Provide a link to the Groq playground with the correct model loaded. Mention the model with parameters you used. Did you use just a system message, or did you use system and user message? What were they? etc.

I used the script at some points on Nebari, but because of it's time wastage, I switched to Groq. But since I'm fully using Groqs right now for all the cheatsheets, I'll delete the scripts.
Right, I'll document how everything about how generated the cheatsheets. I'll create a file for that and push it here.